resources/wikia/modules/flowTracking.js

```diff
@@ -1,7 +1,8 @@
 define('wikia.flowTracking', ['wikia.tracker', 'wikia.window', 'mw', 'jquery'], function (tracker, w, mw, $) {
 
 	var flows = {
-			CREATE_PAGE_DIRECT_URL: 'create-page-direct-url'
+			CREATE_PAGE_DIRECT_URL: 'create-page-direct-url',
+			CREATE_PAGE_REDLINK: 'create-page-redlink'
```

Comment:
I don't think that current implementation distinguishes between article and non-article redlinks